radis/spectrum/spectrum.py

```diff
@@ -3597,6 +3575,117 @@ def normalize(
             print("Normalization factor : {0}".format(norm))
         return out
 
+    # %% Performance / profilers
+
+    def print_perf_profile(self, number_format="{:.3f}"):
+        """Prints Profiler output dictionary in a structured manner.
+
+        Example
+        -------
+        ::
+            Spectrum.print_perf_profile()
+
+            # output >>
```

Comment:
@anandxkumar I updated how the profiler is printed. Here for an example. It shows more explicitly what takes a lot of  time.  Can you review it also work on your side?

Reply:
I pulled the latest code of this PR, on using `Spectrum.print_perf_profile()` prints the profiler correctly, the only issue I was having on my system was the symbol seemed unidentified by the terminal or even Spyder console. I was having the following output:
```
        spectrum_calculation  0.039s â–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆâ–ˆ
        check_line_databank          0.001s 
            reinitialize                 0.003s â–ˆ
            copy_database                0.000s 
            memory_usage_warning         0.003s â–ˆ
            reset_population             0.000s 
```

Same issue on running the new code on the latest develop branch.

Reply:
All fixed ; have a look ! I also updated the docs extensively.